.github/actions/install-wdk/action.yaml

```diff
@@ -109,117 +108,90 @@ runs:
         $inputVersion = '${{ inputs.version }}'
         $versionParts = $inputVersion.Split('.')
         $sdkVersion = $versionParts[0..2] -join '.'
+        $finalVersion= $inputVersion
 
         if ($versionParts.Length -eq 3) {
-            # No QFE specified, find the latest QFE for this base version
-            Write-Host "No QFE specified, searching for latest QFE for version $inputVersion..."
+            Write-Host "QFE not specified, searching for latest QFE for $inputVersion..."
```

Comment:
Another unnecessary change. I suggest avoiding such changes in PRs in the future.